mod/isic/spec/set/self/isic_section_formula_spec.rb

```diff
@@ -10,7 +10,9 @@
 
   describe "integration" do
     it "makes all the calculations between class and section" do
-      create_answer metric: Card["OpenCorporates+Industry Class"], content: %w[0111 9609]
+      create_answer metric: Card["OpenCorporates+Industry Class"],
+                    content: %w[0111 9609],
+                    user: "Joe Admin"
```

Comment:
(This is designer assessed, so after the changes, Joe User couldn't create answers for it any more)